tidy3d/components/source.py

```diff
@@ -737,6 +737,12 @@ class CustomFieldSource(FieldSource, PlanarSource):
         the directionality of the source will likely not be perfect, even if both the ``E`` and ``H`` fields are
         provided. An empty normalizing run may be needed to accurately normalize results.
 
+        To create this empty simulation it is recommended that users create a simulation with no structures but just a flux
+        monitor (``tidy3D.FluxMonitor``) next to the custom source, ensuring that the flux monitor is at least one grid cell
+        away from the source. Moreover, for accurate normalization, users must ensure that the same grid  is used to run
+        the original simulation as well as the empty simulation. The total flux calculated at the flux monitor of the empty
```

Comment:
I think adding a comment about how to achieve the same grid when using auto grid will also be useful. Namely what we did yesterday: define the same structures you used to have in your original Simulation in the GridSpec `override_structures` of the empty sim.

Reply:
Thanks for the feedback! Added a short explanation on how to create empty simulation with the grid from original simulation.